src/meta/processors/partsMan/CreateSpaceProcessor.cpp

```diff
@@ -102,6 +103,10 @@ void CreateSpaceProcessor::process(const cpp2::CreateSpaceReq& req) {
         return;
     }
 
+    if (FLAGS_force_single_part) {
```

Comment:
I think you can set the FLAGS_max_parts_num to 1 instead of add a new flags.

Reply:
No, FLAGS_max_parts_num is for total space partitions, not for single space partition

Reply:
Ok.